Homework_8_classes_objects_fields_methods/MyAmazingRPG/src/Character.java

```diff
@@ -0,0 +1,51 @@
+public class Character {
+
+    static int characterCount;
+
+    public Character() {
+        characterCount++;
+    }
+
+    String name;
+    int health;
+    int strength;
+    int defense;
+    static int potion = 20;
+    int strengthUp;
+    int criticalHit;
+
+    public void healthUp() {
+        this.health = this.health + potion;
```

Comment:
`potion` is static, not a mistake but just wanted to check if you've planned to have only single potion with it's ability to increase health on 20HP for any Character object?